testsuite/tests/basic_symbol_matching/cortex-m/test.py

```diff
@@ -2,9 +2,12 @@
                                     check_symbols, MemoryMap
 
 memmap = MemoryMap()
-memmap.add('flash', 'ROM', 0, 32 * 1024)
-memmap.add('ram', 'RAM', 0x10000, 32 * 1024)
-memmap.add('ccm', 'RAM', 0x20000, 32 * 1024)
```

Comment:
There is no equivalent `ccm` memory region below, is that expected ?

Reply:
I changed this test from using memory region from a real board to just imaginary regions that better fit the need of the test.